react-components/src/architecture/concrete/observationsDomainObject/ObservationsDomainObject.ts

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { CDF_TO_VIEWER_TRANSFORMATION, Overlay3DCollection } from '@cognite/reveal';
+import { OBSERVATION_SOURCE, type ObservationProperties, type Observation } from './models';
+import { VisualDomainObject } from '../../base/domainObjects/VisualDomainObject';
+import { type ThreeView } from '../../base/views/ThreeView';
+import { ObservationsView } from './ObservationsView';
+import { type TranslateKey } from '../../base/utilities/TranslateKey';
+import { type FdmSDK, type InstanceFilter } from '../../../utilities/FdmSDK';
+import { Vector3 } from 'three';
+import { DEFAULT_OVERLAY_COLOR } from './constants';
+import { Changes } from '../../base/domainObjectsHelpers/Changes';
+
+export class ObservationsDomainObject extends VisualDomainObject {
+  private readonly _collection = new Overlay3DCollection<Observation>([], {
+    defaultOverlayColor: DEFAULT_OVERLAY_COLOR
+  });
+
+  public override get typeName(): TranslateKey {
+    return { fallback: ObservationsDomainObject.name };
+  }
+
+  constructor(fdmSdk: FdmSDK) {
+    super();
+    void fetchObservations(fdmSdk).then((observations) => {
+      this.initializeCollection(observations);
+      this.notify(Changes.geometry);
+    });
+  }
```

Comment:
Btw, on classes, in general for easy reading, we usually define the constructor right after the parameters and as Nils mentioned, the override functions on the top (but IMO it should be right below the constructor). At the bottom we keep the rest of the methods (public and then private/protected at the end).
But TBH I dont have a strong opinion on that. As long as we all agree on a unique way to organize the classes, I'm in :)

Reply:
Yes. Please keep the order as I have started with. Otherwise, the navigation is harder!